src/Analysis/Engine/Test/LanguageServerTests.cs

```diff
@@ -992,6 +992,19 @@ class C:
             );
         }
 
+        [TestMethod, Priority(0)]
+        public async Task OnTypeFormatting() {
+            var s = await CreateServer();
```

Comment:
```cs
using(var s = CreateServer()) {
}
```
since it is disposable

Reply:
Mentioned in the other thread on this line, I can do this, but no other test in this file actually disposes of the server either. I can submit a second PR to mass-dispose if you'd like.

Reply:
Closing this in favor of the other one.